analyzer/runtime/runtime.go

```diff
@@ -188,7 +188,7 @@ func (m *Main) latestRound(ctx context.Context) (uint64, error) {
 		m.qf.LatestBlockQuery(),
 		// ^analyzers should only analyze for a single chain ID, and we anchor this
 		// at the starting round.
-		EmeraldDamaskAnalyzerName,
+		SapphireDamaskAnalyzerName,
```

Comment:
bit of inconsistency with some recent hurried runtime db table generalization. a lot of our queries right now do string interpolation within the query factory with `qf.runtime`, whereas these leave the runtime parametric. let's choose one and make them uniform.

1. qf knows runtime, analyzer doesn't need to provide it. update these to remove the runtime sql placeholder and convert to a format string thing in query factory.
2. analyzer knows runtime, qf doesn't care. update runtime queries to use a sql placeholder and have analyzer pass it. qf no longer needs `runtime` field.

I'm voting 2

Reply:
This query is a tiny bit different from others because 
- it includes the `consensus_damask` analyzer: https://github.com/oasisprotocol/oasis-indexer/blob/699e4cccbde673c2e61bbc25822401512e756db4/analyzer/consensus/consensus.go#L202-L202
- the column technically refers to the analyzer, not the runtime. I can see a scenario (though I think it's unlikely) where we want to do some additional processing on already-processed blocks, and we'll whip up a temporary analyzer that will also track its progress in this table.

Reply:
hmh

Reply:
ok let's leave `.runtime` in QueryFactory for now